std_msgs/Header header
vehicle_platform/EsrTrackMotionPowerId[] trackpower